homeassistant/components/websocket_api/http.py

```diff
@@ -365,28 +366,28 @@ def handle_hass_stop(event: Event) -> None:
                         break
                     handler = msg.data[0]
                     payload = msg.data[1:]
-                    connection.async_handle_binary(handler, payload)
+                    async_handle_binary(handler, payload)
                     continue
 
                 if msg.type != WSMsgType.TEXT:
                     disconnect_warn = "Received non-Text message."
                     break
 
                 try:
-                    msg_data = msg.json(loads=json_loads)
+                    command_msg_data = json_loads(msg.data)
```

Comment:
This was typed to Any so we lost typing when go through here, but `.data` is available (as used in the binary handler)